### PR TITLE
Update The Graph docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR is an extension of #944. It cleans up the docs to meet our standards.

⚠️ Note: screenshots are coming soon! ⚠️

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [x] If this page requires a disclaimer, I have added one

### CODE REVIEW

File: builders/integrations/indexers/thegraph.md [hunk 1]
```diff
@@ -5,319 +5,172 @@ description: Learn how to build APIs, called subgraphs, to store and fetch on-ch
...
 # Using The Graph on Moonbeam
 
-## Introduction {: #introduction }
+## Introduction  {: #introduction }
 
-Indexing protocols organize information in a way that applications can access it more efficiently. For example, Google indexes the entire internet to provide information rapidly when you search for something.
+Indexing protocols organize information so that applications can access it more efficiently. For example, Google indexes the entire Internet to provide information rapidly when you search for something.
 
-[The Graph](https://thegraph.com){target=\_blank} is a decentralized and open-source indexing protocol for querying networks like Ethereum. In short, it provides a way to efficiently store data emitted by events from smart contracts so that other projects or DApps can access it easily.
+[The Graph](https://thegraph.com/){target=\_blank} is a decentralized and open-source indexing protocol for querying networks like Ethereum. In short, it provides a way to efficiently store data emitted by events from smart contracts so that other projects or DApps can access it easily.
```
Comment (dawnkelly09) [inline builders/integrations/indexers/thegraph.md:12]: DApps should be dApps per style guide

**Review by dawnkelly09 — CHANGES_REQUESTED** [review]
re the suggestion: if user misses the instructions to cd in the terminal (which I did initially...it's a ton of lines flying past!) the rest of the demo work properly.

**Comment by eshaben** [comment]
@dawnkelly09 done!

**Review by dawnkelly09 — APPROVED** [review]
lgtm

**Pull request merged**